regulations/static/regulations/css/less/module/comment.less

```diff
@@ -467,8 +467,9 @@ a.comment-index-review {
 
   .comment-attachments {
     display: block;
+    float: none;
```

Comment:
Looks like this is intentionally floated around line 365 as of #325. Would you mind verifying that the original problem is still fixed and, if so, just remove `.comment-attachments` from that list?